ElectronApp/src/renderer/components/makeData.tsx

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-plusplus */
+const range = (len: number) => {
+  const arr = [];
+  for (let i = 0; i < len; i++) {
+    arr.push(i);
+  }
+  return arr;
+};
+
+const newTree = () => {
```

Comment:
This is good for demo purposes, but we probably don't need to be pre-populating our cells with data.